duckduckgo/duckduckgo-privacy-extension
# 🧼 Remove img/ from web_accessible_resources

**Reviewer:** @dharb 

## Description:
Legacy clean up. This PR removes `img/` folder from `web_accessible_resources`, so that they no longer can be requested by sites. Additionally, all images that are not referenced anywhere in code were removed.

## Steps to test this PR:
Click around the popup and the settings page to make sure that no images are missing.

## Automated tests:

###### Reviewer Checklist:

###### PR Author Checklist:
